include/swift/Option/Options.td

```diff
@@ -890,7 +890,11 @@ def no_static_executable : Flag<["-"], "no-static-executable">,
 
 def use_ld : Joined<["-"], "use-ld=">,
   Flags<[DoesNotAffectIncrementalBuild]>,
-  HelpText<"Specifies the linker to be used">;
+  HelpText<"Specifies the flavor of the linker to be used">;
+
+def ld_path : Joined<["--"], "ld-path=">,
```

Comment:
Why the `--` form rather than `-` form?

Reply:
I wanted to match the spelling in Clang, which also simplifies forwarding the option; though I don't have a strong preference so I can change this to a single dash if you'd prefer.